src/app.css

```diff
@@ -3,15 +3,14 @@
   font-size: 16px;
   line-height: 24px;
   font-weight: 400;
+  background: url('./bg.png') no-repeat center;
+  background-size:cover;
+  height:100vh;
```

Comment:
Please change `height` to `min-height` so that is doesn't break when the page has scroll
![image](https://user-images.githubusercontent.com/49235295/197449759-11cd737d-8913-4e00-9fe7-d5cd2cb14328.png)